rkt/api_service.go

```diff
@@ -395,6 +395,16 @@ func (s *v1AlphaAPIServer) ListPods(ctx context.Context, request *v1alpha.ListPo
 	if err := walkPods(includeMostDirs, func(p *pod) {
 		pod, manifest := getBasicPod(p)
 
+		flavor, err := os.Readlink(filepath.Join(p.path(), "/stage1/rootfs/flavor"))
```

Comment:
Using this path is not documented in https://github.com/coreos/rkt/blob/master/Documentation/devel/stage1-implementors-guide.md
So there is little guarantee that future stage1s will still have this file.

I thought there was another way to get the stage1 flavor... let me have a look.